### ISSUE TITLE
Driver closing is not working so well

### ISSUE THREAD

**Issue by claudia-lat** [robotology-playground/forcetorque-yarp-devices#3]
I found that the closing of the driver is not straightforward.

**Comment by francesco-romano** [comment]
@claudia-lat 

What do you mean with "straightforward" ?
Which driver (as this repository contains 3 different drivers)?

**Comment by claudia-lat** [comment]
The AMTI one. If you try to close the driver it gives you an error.

**Comment by francesco-romano** [comment]
If you are able to add a log to the issue, it would be helpful

**Comment by claudia-lat** [comment]
Error message:

[AnIaNloFgWOra]pApelr:l  aAncaltoigoSnesr vfeorr:  aScetnisoonr  lreevteulr n1e0d  oefr rsohru twdiotwhn  cpohdaes e1 .finished.
[ERROR]AnalogWrapper: AnalogServer: Sensor returned error with code 1.

**Comment by randaz81** [comment]
LOL! (That race on the message is known and it has to be fixed)

**Comment by randaz81** [comment]
No, I was referring to the wrong display of the printed message only, not to the returned error code.

**Comment by DanielePucci** [comment]
So "not straightforward" = "error" -.-
